.github/workflows/android.yml

```diff
@@ -77,6 +78,49 @@ jobs:
           if-no-files-found: ignore
           path: ${{ runner.temp }}/artifacts/
 
+  # Running Android emulator directly on the runner and not using Docker
+  run-emulator:
+    needs: build-llm-demo
+    runs-on: amz2023.linux.4xlarge
+    env:
+      ANDROID_NDK_VERSION: r26c
+    steps:
+      - name: Setup SSH (Click me for login details)
+        uses: pytorch/test-infra/.github/actions/setup-ssh@main
+        if: ${{ !contains(matrix.runner, 'gcp.a100') }}
```

Comment:
Why do we need to check if the runner is with nVidia A100? I think for ExecuTorch we don't need such a runner. Would it be possible to mistakenly allocated to ExecuTorch?

Reply:
I think it's "if the task does not specify requiring gcp.a100"?

Reply:
Agree that we may not need to check this at all

Reply:
Agree.  For the context, ssh is only available on AWS linux runner, it doesn't work on GCP Linux runner, thus the check, and we don't need it here